docs/source/modules.rst

```diff
@@ -0,0 +1,7 @@
+:mod:`ODMconnection`
```

Comment:
This is only an example. We can add all the modules later once this is working.

Reply:
Got it